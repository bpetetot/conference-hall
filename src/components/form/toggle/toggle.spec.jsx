/* eslint-env jest */
import React from 'react'
import Toggle from './index'

const props = {
  name: 'name',
  onChange: jest.fn(),
}

describe('components/toggle', () => {
  it('should render', () => {
    expect(<Toggle {...props} />).toMatchSnapshot()
  })
})
