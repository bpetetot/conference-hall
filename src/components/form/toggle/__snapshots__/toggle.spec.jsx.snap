// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/toggle should render 1`] = `
<Toggle
  checked={false}
  name="name"
  onChange={[MockFunction]}
/>
`;
